infrastructure/terraform/environments/development-eu-west-2/devops1/vars

```diff
@@ -14,16 +16,8 @@ ec2_instance_type = "t3.medium"
 # Certificate for https access through ALB
 ssl_certificate_id = "arn:aws:acm:eu-west-2:169942020521:certificate/248f5001-3d30-4d90-bec3-023e5ab1f572"
 zone_id = "Z2KSI4Z5ZN9NT0"
-zone_name = "aws.chdev.org"
-
-
-//---------------- START: Environment Secrets for services ---------------------
-
-secrets = {
-  secret_mongo_user_name  = "AQICAHiujiSaeV3dDdEKHuZ/9qHGe7ylBwQFW/vMFainoOEy5wEU10eVcDR2a7QDqB7Qtp7LAAAAYzBhBgkqhkiG9w0BBwagVDBSAgEAME0GCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMaIkaIg/tmYqzwC3RAgEQgCBufCUdnVfYNxlu8lVOZg5Z/j0viEuRTAMk/2ZjuCr/nw=="
-  secret_mongo_password   = "AQICAHiujiSaeV3dDdEKHuZ/9qHGe7ylBwQFW/vMFainoOEy5wEU10eVcDR2a7QDqB7Qtp7LAAAAYzBhBgkqhkiG9w0BBwagVDBSAgEAME0GCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMaIkaIg/tmYqzwC3RAgEQgCBufCUdnVfYNxlu8lVOZg5Z/j0viEuRTAMk/2ZjuCr/nw=="
-}
-
-//---------------- END: Environment Secrets for services ---------------------
+#top_level_domain = ".devops1.aws.chdev.org"
```

Comment:
We shouldn't have any commented out code

Reply:
Removed.